app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -137,7 +125,7 @@ const AnalysisUnavailable = ({
   error,
 }: {
   experiment: getExperiment_experimentBySlug;
-  error: Error | undefined;
+  error: boolean;
```

Comment:
@LZoog with this change we no longer pass an actual error down to PageResults, but in practice the absence of analysis data (remember: even if the analysis isn't ready, the data os still returned) is indicative of an error.

Reply:
Couldn't it just be unavailable but not error? That's why I've got `{error ? <AlertError /> : <AlertUnavailable />}`. 🤔 When we do the route protection stuff it shouldn't matter either way so meh either way.

Reply:
If it's unavailable the request would still complete and the returned object would contain `show_analysis: false`. But if `analysis` is straight up `undefined` that's an error.